### PR TITLE
Currency:Apicurio reg patch

### PR DESCRIPTION
**Pull request by stutiibm**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: a/apicurio-registry/build_info.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
+    "package_name" : "apicurio-registry",
+    "github_url":"https://github.com/Apicurio/apicurio-registry",
+    "version": "2.4.1.Final",
+    "package_dir": "a/apicurio-registry",
+    "build_script": "apicurio-registry_ubi_8_5.sh",
+    "validate_build_script": "true",
+
+
+    "2.4.1.Final": {
```
Comment (nikhil-kalbande) [inline a/apicurio-registry/build_info.json:10]: @stutiibm  I dont see you are using this "PATCH" variable in build script. Do we really need this. Can we not just use "PACKAGE_VER" variable?

File: a/apicurio-registry/apicurio-registry_ubi_8_5.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+    exit 1
+fi
+
+if ! mvn test; then
+    echo "------------------$PACKAGE_NAME:install_success_but_test_fails---------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_success_but_test_Fails"
+    exit 1
```
Comment (nikhil-kalbande) [inline a/apicurio-registry/apicurio-registry_ubi_8_5.sh:67]: @stutiibm  Please make it to exit 2

File: a/apicurio-registry/build_info.json (hunk 1 above)
Comment (stutiibm) [reply a/apicurio-registry/build_info.json]: Removed PATCH variable.

File: a/apicurio-registry/apicurio-registry_ubi_8_5.sh (hunk 2 above)
Comment (stutiibm) [reply a/apicurio-registry/apicurio-registry_ubi_8_5.sh]: Made required changes.

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**